tests/test_moment_matching.py

```diff
@@ -0,0 +1,74 @@
+import tensorflow as tf
+import tensorflow_probability as tfp
+tfd = tfp.distributions
+
+from pilco.policies import RBFPolicy
+
+
+class MomentMatchingError(Exception):
+    pass
+
+
+def test_mm_rbf_policy():
```

Comment:
We talked about maybe renaming these to EQ or SE later? This is not burning, but probably it should be uniform across the library.

Reply:
Yeah, I agree with this, we should rename RBFPolicy. Since this test tested RBFPolicy, I called it that and will change the name once we clean and rename RBFPolicy.